cmd/charm/charmcmd/grant.go

```diff
@@ -49,8 +49,7 @@ The --set parameters is used to overwrite any existing ACLs for the charm or bun
     charm grant ~johndoe/wordpress --acl write --set fred,bob
 
 When a specific charm or bundle revision is provided,
-the channel parameter is ignored. Otherwise the "stable"
-channel is used by default. To select another channel,
+the channel parameter is ignored. To select a channel,
```

Comment:
That's not quite right either. The channel is still used when a revision is provided, but if a revision is provided but no channel is specified, the default channel for that revision of the entity is chosen.